src/Components/Projects/Form/index.js

```diff
@@ -8,181 +9,141 @@ import FunctionalButton from '../../Shared/Buttons/FunctionalButton';
         const data = await response.json();
-        console.log('project data', data);
-        setProjectBody({
+        console.log(data);
+        setValues({
           name: data.data.name,
           clientName: data.data.clientName,
           description: data.data.description,
           startDate: data.data.startDate.substr(0, 10),
           endDate: data.data.endDate.substr(0, 10),
+          employees: data.data?.employees.map((employee) => {
+            return {
+              rate: employee.rate,
+              role: employee.role,
+              employee: employee.employee?._id
+            };
+          }),
           active: true
         });
-        setEmployees(data.data.employees.filter((item) => item.employee !== null));
       } catch (error) {
         console.error(error);
       }
       dispatch(setFetching(false));
     }
   }, []);
 
-  const addProject = (body) => {
-    dispatch(createProject(body));
-    setModalDisplay(true);
-  };
-
-  const changeProject = (body) => {
-    dispatch(editProject(id, body));
-    setModalDisplay(true);
-  };
-
-  const onSubmit = (e) => {
-    e.preventDefault();
-
-    const unpopulatedEmployees = employees.map((emp) => {
-      return { role: emp.role, rate: emp.rate, employee: emp.employee._id };
-    });
-
-    const body = { ...projectBody, employees: unpopulatedEmployees };
-    !isValidId ? addProject(body) : changeProject(body);
-  };
-
-  const onChange = (e) => {
-    setProjectBody({ ...projectBody, [e.target.name]: e.target.value });
-  };
-
-  const onChangeEmployee = (event, employee) => {
-    setEmployees((employees) => {
-      return employees.map((emp) => {
-        if (emp.employee._id === employee.employee._id) {
-          return {
-            employee: {
-              ...employee.employee,
-              _id: event.target.name === 'id' ? event.target.value : employee.employee._id
-            },
-            role: event.target.name === 'role' ? event.target.value : employee.role,
-            rate: event.target.name === 'rate' ? event.target.value : employee.rate
-          };
-        }
-        return emp;
-      });
-    });
-  };
-
   useEffect(() => {
-    console.log('project state', projectBody);
-  }, [projectBody]);
+    reset(values);
+  }, [values]);
 
-  useEffect(() => {
-    console.log('employees state', employees);
-  }, [employees]);
+  const addProject = (data) => {
+    dispatch(createProject(data));
+  };
 
-  const deleteEmployee = (employee) => {
-    setEmployees((employees) => {
-      return employees.filter((emp) => emp.employee._id !== employee.employee._id);
-    });
+  const changeProject = (data) => {
+    dispatch(editProject(id, data));
   };
 
-  const addEmployee = () => {
-    setEmployees(() => {
-      return [...employees, { employee: {}, rate: 0, role: '' }];
-    });
+  const onSubmit = async (data) => {
+    console.log(data);
```

Comment:
```suggestion
    console.log(data);
```
Deletion also requested for this one